### PR TITLE
feat: services filter

### PR DESCRIPTION
**Pull request by amanuela97**
As a user, I would like to filter services based on the city and the service type. see https://trello.com/c/TnFo7mCd/310-filter-by-city & https://trello.com/c/5To9j8Z2/311-filter-by-service

## Acceptance Criteria
- [x] dropdown for locations is present
- [x] dropdown for services is present
- [x] dropdowns filter as expected
- [x] page route corresponds to the filter

## Demo
https://deploy-preview-77--wehelpvip.netlify.app/services/

### CODE REVIEW

**Review by VOlekh — COMMENTED** [review]
Amanual, for me 2 points don't work as expected: dropdowns filter as expected
AND page route corresponds to the filter. I'll add screenshots in slack. 1) If I change place for the 2d time the URL query appends to the previous value https://deploy-preview-77--wehelpvip.netlify.app/services/Berlin/Accomodation/Dresden/Accomodation. 2) correct sequence in URL service-> time-> place, so I would change button order too 3) even with a first choice I have cards with different places on a screen like I choose Berlin but see: anywhere, Munchen, Berlin, Ukrain, Viena

File: components/layout/header.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@
     >
       <PopoverPanel
         focus
-        class="bg-white-500 h-auto absolute top-0 inset-x-0 transition transform origin-top-right md:hidden overflow-hidden"
+        class="bg-white-500 z-10 h-auto absolute top-0 inset-x-0 transition transform origin-top-right md:hidden overflow-hidden"
```
Comment (VOlekh) [inline components/layout/header.vue:46]: what do you need z for?

File: components/layout/footer.vue [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <template>
-  <footer class="bg-white-500" aria-labelledby="footer-heading">
+  <footer
```
Comment (VOlekh) [inline components/layout/footer.vue:2]: why did you change the footer?

Comment (amanuela97) [reply components/layout/footer.vue:2]: I added a border-top to the footer to separate it from the rest of the page

File: components/layout/header.vue (hunk 1 above)
Comment (amanuela97) [reply components/layout/header.vue:46]: so the drop-down menu does not overlap with the page content. This should also fix the partners logo overlapping it

File: pages/services/index.vue [hunk 3]
```diff
@@ -16,58 +42,82 @@ import { fetchServices } from '../../utils/airtable'
...
-  data: () => ({
-    services: [],
-  }),
+  data() {
+    const params = this.$route.params
+    const loc = this.capitalizeFirstLetter(params.location)
+    const serv = this.capitalizeFirstLetter(params.service)
+    return {
+      services: null,
+      location: loc || 'Anywhere',
```
[the remark this answers is not in the document]
Comment (VOlekh) [reply pages/services/index.vue:51]: I mean if you choose only Anywhere in the dropdown. Or we need All.

**Review by VOlekh — APPROVED** [review]
Amanual, I think the filter doesn't work properly. I took one case to find Legal Aid. To find this type of service is impossible if you don't know the country. We need category All which will include all the countries we have +Anything+Empty. Take a look we have lots of records with empty locations. you would only find Legal Aid if you enter Poland or Viena. The same for services. Should see All services in a chosen country. Please ask Alex if this case is required. No comments to the code from my side so can be merged if we agree on the search criteria.
